src/tensors/vectors.jl

```diff
@@ -0,0 +1,23 @@
+struct OffByOneVector{T} <: AbstractVector{T}
+    data::AbstractVector{T}
+end
+
+function Base.getindex(vector::OffByOneVector{T}, index::Int) where {T}
+    return vector.data[index] + 1
+end
+
+function Base.getindex(vector::OffByOneVector{T}, index::Vararg{Int}) where {T}
+    return vector.data[index...] + 1
+end
+
+function Base.setindex!(::OffByOneVector{T}, ::Any, ::Int) where {T}
```

Comment:
let's make these mutable so we can write to a python-readable object and convert back if we need to

Reply:
Done!